platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/MapClickEvent.java

```diff
@@ -0,0 +1,122 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import android.content.Context;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.android.telemetry.Event;
+import com.mapbox.android.telemetry.TelemetryUtils;
+
+class MapClickEvent extends Event implements Parcelable {
+  private static final String MAP_CLICK = "map.click";
+
+  @SerializedName("event")
```

Comment:
can we remove `@SerializedName` if the field has the same name as the serialized name?